<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_order_formation</class>
 <widget class="QDialog" name="Dialog_order_formation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Формирование заказа</string>
  </property>
  <widget class="QPushButton" name="btn_req">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Сформировать</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_code">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>10</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Код пробирки</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_surname">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Фамилия</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Имя</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_middle_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Отчество</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_service">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Услуга</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
